src/jsifier.js

```diff
@@ -45,7 +47,8 @@ function JSify(data, functionsOnly) {
     // else. This lets us not hold any strings in memory, we simply print
     // things out as they are ready.
 
-    var shellParts = read(shellFile).split('{{BODY}}');
+    var thirdParty = thirdPartyFiles.map(function(f) { return read(f) }).join('\n');
+    var shellParts = read(shellFile).replace('{{THIRD_PARTY}}', thirdParty).split('{{BODY}}');
```

Comment:
This seems to add a third party library sodium to all generated builds? That is awkward for build sizes, plus also it will trigger a legal review for a number of partner companies, which we would prefer to avoid. It looks like sodium is used to base64 decode? Could https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding be used instead?

This looks like a good feature, but it should definitely be implemented as a build option so that builds which don't use it don't need to compromise on generated build sizes. That is, gate the generated code on `SINGLE_FILE` option to not impact builds which don't utilize `SINGLE_FILE`?